ui-library-starting-with-chat-stateful/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@azure/communication-calling": "1.16.3-beta.1",
+    "@azure/communication-calling": "1.17.1-beta.5",
```

Comment:
As Node 16 is a recommended version for this package, peer dependencies for `@azure/communication-react` can be skipped here
https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/#

Reply:
We need to review the other dependencies and update across the board. We should be using node 18+ now (LTS version).